### DISCUSSION TITLE
Add an option to use ISO 8601 date string as migration name prefix

### DISCUSSION THREAD

**Discussion by jkazimierczak** [tmm/kysely-migrate#21] [Ideas]
Kysely docs suggest to use ISO 8601 date string for prefixing migration names, as presumably this works better for larger teams. Citing the docs:
> A date prefix works well in large teams where multiple team members may add migrations at the same time in parallel commits without knowing about the other migrations.
> 
> Source: https://kysely.dev/docs/migrations#execution-order

An toggle/flag option could be added to the `create` command. This can be done simply with just adding a condition to [`create.ts`](https://github.com/tmm/kysely-migrate/blob/aab0e2ff73593da297941fc2b046faa3625dac4d/src/commands/create.ts#L27). I'm up for help and can submit a PR if you want.

**Comment by tmm** [comment]
Feel free to submit a PR!
